src/command_spec.rs

```diff
@@ -15,7 +15,7 @@ impl Display for CommandSpec {
         let env_str = self
             .envs
             .iter()
-            .map(|(k, _)| format!("{k}=***"))
```

Comment:
This should probably be reviewed by the security team instead?

Reply:
Why? Generally, it is OK to log environment variables. The ones which specifically should not be logged should go to the `secret_env_vars` field. But we certainly want to log for example `ROBOCORP_HOME` or `RCC_REMOTE`.

Reply:
done